framework/include/security/security_common.h

```diff
@@ -200,6 +200,10 @@ typedef struct _security_ecdh_param {
  */
 security_error security_init(security_handle *hnd);
 security_error security_deinit(security_handle hnd);
+#ifdef CONFIG_ARM_TRUSTZONE
+security_error security_allocate(security_handle hnd, uint32_t ssize);
```

Comment:
These are related to arm trustzone. Why don't you rename to something to represent trustzone?

Reply:
these apis are needed when secure element functions/data will be moved to trustzone image to make it secure. Either we re-name all secure element apis with trustzone, or we keep these additional apis as security_*  to maintain consistency.